source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRichTextBlockRenderer.mm

```diff
@@ -75,55 +75,91 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
+                                               range:selectActionRange];
+                        [textRunContent addAttribute:NSUnderlineColorAttributeName
+                                               value:foregroundColor
+                                               range:selectActionRange];
                     }
                 }
 
                 // apply hightlight to textrun
-                if(textRun->GetHighlight()) {
+                if (textRun->GetHighlight()) {
                     UIColor *highlightColor = [acoConfig getHighlightColor:style
                                                            foregroundColor:textRun->GetTextColor()
                                                               subtleOption:textRun->GetIsSubtle()];
-                    [textRunContent addAttribute:NSBackgroundColorAttributeName value:highlightColor range:NSMakeRange(0,textRunContent.length)];
+                    [textRunContent addAttribute:NSBackgroundColorAttributeName
+                                           value:highlightColor
+                                           range:NSMakeRange(0, textRunContent.length)];
                 }
-                
-                if(textRun->GetStrikethrough()) {
-                    [textRunContent addAttribute:NSStrikethroughStyleAttributeName value:[NSNumber numberWithInteger:NSUnderlineStyleSingle] range:NSMakeRange(0, textRunContent.length)];
+
+                if (textRun->GetStrikethrough()) {
+                    [textRunContent addAttribute:NSStrikethroughStyleAttributeName
+                                           value:[NSNumber numberWithInteger:NSUnderlineStyleSingle]
+                                           range:NSMakeRange(0, textRunContent.length)];
                 }
-                
-                if(textRun->GetUnderline())
-                {
-                    [textRunContent addAttribute:NSUnderlineStyleAttributeName value:[NSNumber numberWithInteger:NSUnderlineStyleSingle] range:NSMakeRange(0, textRunContent.length)];
+
+                if (textRun->GetUnderline()) {
+                    [textRunContent addAttribute:NSUnderlineStyleAttributeName
+                                           value:[NSNumber numberWithInteger:NSUnderlineStyleSingle]
+                                           range:NSMakeRange(0, textRunContent.length)];
                 }
 
-                // Add paragraph style, text color, text weight as attributes to a NSMutableAttributedString, content.
-                [textRunContent addAttributes:@{NSParagraphStyleAttributeName:paragraphStyle, NSForegroundColorAttributeName:foregroundColor,} range:NSMakeRange(0, textRunContent.length - 1)];
+                // Add paragraph style, text color, text weight as attributes to a
+                // NSMutableAttributedString, content.
+                [textRunContent addAttributes:@{
+                    NSParagraphStyleAttributeName : paragraphStyle,
```

Comment:
>: [](start = 50, length = 1)

nit: also here the ':' are not aligned

Reply:
same here, it's the result of running clang format.

---
In reply to: [322860767](https://github.com/microsoft/AdaptiveCards/pull/3430#discussion_r322860767) [](ancestors = 322860767)